### PR TITLE
BUILD-261: Add release version to install instructions

### PR DESCRIPTION
**Pull request by adambkaplan**
- Update the install instructions to use the current dev preview release
  tag for the release.yaml URL.
- Restructure the install instrucitons to suggest the release.yaml
  approach first.
- Update the validation instructions.

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -6,7 +6,33 @@
 
 1. Grant `cluster-admin` permissions to the current user.
 
-## Installing from a local clone of this repository (only developer preview level support)
+## Installation (developer preview)
+
+### Installing from the release page
+
+1. Run the following command
+
```
Comment (gabemontero) [inline docs/install.md:14]: add a link to https://github.com/openshift/csi-driver-projected-resource/releases so then know what are their options for setting RELEASE_VERSION

**Comment by gabemontero** [comment]
/retitle BUILD-261: Add release version to install instructions

File: docs/install.md (hunk 1 above)
Comment (adambkaplan) [reply docs/install.md:15]: updated

**Pull request merged**